xl_auth/templates/public/home.html

```diff
@@ -1,39 +1,64 @@
 <!-- public/home.html -->
 {% extends "layout.html" %}
 {% block content %}
-    <!-- Main jumbotron for a primary marketing message or call to action -->
-    <div class="jumbotron">
-
-        <h1>{{ _('Welcome to xl_auth') }}</h1>
-
-        <p>This is a starter Flask template. It includes Bootstrap 3, jQuery 2, Flask-SQLAlchemy,
-            WTForms, and various testing utilities out of the box.</p>
-        <p><a href="https://github.com/sloria/cookiecutter-flask"
-              class="btn btn-default btn-large">Learn more &raquo;</a></p>
-    </div><!-- /.jumbotron -->
-
     <div class="body-content">
         <div class="row">
+          {% if current_user and current_user.is_authenticated %}
+          <div class="jumbotron">
+
+              <h1>{{ _('Welcome to xl_auth') }}</h1>
+
+              <p>This is a starter Flask template. It includes Bootstrap 3, jQuery 2, Flask-SQLAlchemy,
+                  WTForms, and various testing utilities out of the box.</p>
+              <p><a href="https://github.com/sloria/cookiecutter-flask"
+                    class="btn btn-default btn-large">Learn more &raquo;</a></p>
+          </div><!-- /.jumbotron -->
+          {% elif login_form %}
+            <div class="col-md-12 login-panel panel panel-default">
+              <div class="col-md-8">
+                <div class="header">
+                  <div class="logo-small logotype logotype-black">
+                  </div>
+                  <h2>
+                    xl_auth
+                  </h2>
+              </div>
+                <p>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec porta quis dolor sed hendrerit. Maecenas varius augue vitae quam molestie condimentum. Pellentesque nec magna accumsan, placerat dui ut, tristique nulla.
+                </p>
+                <p>
+                  Nam commodo velit libero, ac lacinia tellus ornare a. Pellentesque magna arcu, consectetur eu orci eu, maximus ullamcorper turpis. Vivamus lobortis tellus non nibh vulputate, at rhoncus est vehicula.
+                </p>
+              </div>
+              <div class="col-md-4">
+                <h2>{{ _('Log in') }}</h2>
```

Comment:
@oBlissing We Pythonistas 🐍 are is all about style and consistency.  Try making the tiniest code changes (like extra blank line after docstring) and run `flask lint` and you'll see what I mean. Let's apply the same consistency in templates as well!

Can you run PyCharm *re-format code* on the updated templates and commit?

![screen shot 2017-10-16 at 2 30 08 pm](https://user-images.githubusercontent.com/786326/31612160-20eaf3ea-b27f-11e7-9c29-0a9ff71e63a5.png)